src/util/aquarius.ts

```diff
@@ -129,26 +166,32 @@ export function getSearchQuery(
 
 export async function searchMetadata({
   term,
-  type,
-  from
+  from,
+  tag,
+  size,
+  sortBy,
+  sortDirection,
+  type
 }: {
   term: string
-  type?: MetadataMain['type']
   from?: number
+  tag?: string
+  size?: string
+  sortBy?: SortByOptions
+  sortDirection?: Sort['type']['order']
+  type?: MetadataMain['type']
 }): Promise<SearchResponse | undefined> {
   try {
     const searchQuery = {
-      ...getSearchQuery(term, type),
+      ...getSearchQuery(term, tag, size, sortBy, sortDirection, type),
       from: from || 0
     }
 
-    //console.log(`Query ${apiBasePath} for ${term}:`, searchQuery)
+    console.log(`Query ${apiBasePath} for ${term}:`, searchQuery)
```

Comment:
chore(blocking): make sure to remove the comment later. Add a `TODO` to remind us.